### ISSUE TITLE
Trying EfficientDet-D3

### ISSUE THREAD

**Issue by alexiniesta7** [signatrix/efficientdet#23]
Hello! I am trying to train EfficientDet-D3. Could you, please, tell me the changes I have to do? I've already change model = EffNet.from_pretrained('efficientnet-b3'), compound_coef=0 to compound_coef=3 and image_size=896, and I've obtained the following error:
RuntimeError: Given groups=1, weight of size 160 40 1 1, expected input[10, 48, 64, 64] to have 40 channels, but got 48 channels instead.
So I have also needed to change the output channels of the convolutions from 40 to 48, from 80 to 96 and from 192 to 232 and it seems to work.
Am I missing something?
Thanks in advance :)

**Comment by lchunleo** [comment]
Any solutions to have other than b0 to train?

**Comment by lchunleo** [comment]
How to solve it? Any help?

**Comment by alexiniesta7** [comment]
With the changes I mentioned it works and train correctly, but I am not able to achieve the performance mentioned on the paper

**Comment by Slimco86** [comment]
Yes, I also had to change the amount of input channels manually, since this part of scaling is apparently not implemented. To automate it, the scaling factor has to be derived, or simply infer the output number of channels of the backbone nodes and put it into the input number of channels.
